src/sessions/base-session-handler.ts

```diff
@@ -132,7 +133,13 @@ export default abstract class BaseSessionHandler {
    * @param options for updating outgoing media
    */
   async updateOutgoingMedia (session: IJingleSession, options: IUpdateOutgoingMedia): Promise<any> {
-    this.log(LogLevels.info, 'updating outgoing media', { conversationId: session.conversationId, options });
+    this.log(LogLevels.info, 'updating outgoing media', {
```

Comment:
This was one of the causes for `413`s on logs. It was logging the entire jingle session.